ethernet_init/idf_component.yml

```diff
@@ -1,8 +1,8 @@
 dependencies:
   idf:
-    version: '>=5.0'
+    version: '>=5.1'
```

Comment:
why we not support 5.0 now?

Reply:
Because the poll mode for SPI ETH modules is supported since v5.1.

Reply:
@kostaond I see, If we have made a change to the minimum supported version of the idf, @bogdankolendovskyy will we consider increasing the version to 0.2.0? This is more user-friendly for users who use 5.0 IDF and specify `ethernet_init` version of 0.1.* (commonly used, to use latest bugfix version)

Reply:
Updated as suggested.